OneCommunityGlobal/HighestGoodNetworkApp
# Lucile - add work confirmation improvements to people report

###Description
9. (PRIORITY MEDIUM) Jae: Add work confirmation improvements to People Report (WIP Lucile)
Admin or Owner login → Reports → Reports → People → Choose
Remove “hours logged this week” section: If a user is INACTIVE (has an end date), then the “Hours Logged This Week” square should not show. Center the remaining ones.
Please also add the User’s Title under their name.
Please also add a “Rehireable” section under their name that can be clicked as Yes or No

11. (PRIORITY LOW): In smaller view widths (<=1200px), the tops of the report blocks are cut off. (WIP Lucile)

###Related PR
To test this frontend PR you need to checkout the  #362 backend PR.
https://github.com/OneCommunityGlobal/HGNRest/pull/362

###Mainly changes explained:
1 - Was added a new set State in setRehireable in PeapleReport

###Before (sorry for my slow connexion)

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/111132148/e92b0a4c-9f38-4129-963b-645cdada4b9d

###After (sorry for my slow connexion)


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/111132148/ae80a0ca-568f-454f-93f5-adfaa0bd0eef





###How to test:
Git checkout Lucile_add_work_confirmation_improvements_to_people_report in the frontend
do npm install and npm run start:local to run this PR locally

Git checkout Lucile_add_work_confirmation_improvements_to_people_report in the backend
do npm install and npm run build and npm start



Admin or Owner login → Reports → Reports → People → Choose someone with an end date 
You should not see "hours logged this week" section
You should see the User’s Title under their name
You should be able to checkout and checkout the rehireable check box on the right.
The chosen check should remain even if the page is refreshed.
The tops of the report blocks should not be cut off in smaller views

Admin or Owner login → Reports → Reports → People → Choose someone without an end date 
You should see "hours logged this week" section
You should see the User’s Title under their name
You should not see the rehireable checkbox
The tops of the report blocks should not be cut off in smaller views
